docs/_data/jekyll_filters.yml

```diff
@@ -18,16 +18,19 @@
 #
 - name: Relative URL
   description: >-
-    Prepend the <code>baseurl</code> value to the input. Useful if
-    your site is hosted at a subpath rather than the root of the domain.
+    Transforms a file path into the corresponding relative url by prepending the 
```

Comment:
```suggestion
    Transforms a file path into the corresponding relative URL by prepending the 
```

Reply:
This is not accurate to say **file path**.

Then it seems like you would pass `'_my-dir/my-page.md'` to it.

When you would really pass a value like `/my-dir/my-page.html` which is a **URL** path not a file path.

Common usage to illustrate this:

```
{{ page.url | relative_url }}
```

```
input: /my-dir/my-page.html
output: /my-repo/my-dir/my-page.html
```

Also the description is verbose. Rather, `convert` or `make`. And keep the voice active with `prepend` rather than `prepends` or `transforms` which makes it drag.

```
Prepend <code>baseurl</code> value to the input to make a URL into a relative URL. This is recommend for a site that is hosted on a subpath of a domain.
```

Also I've changed useful to recommended - as if you have a site that is hosted on a subpath then you will probably want to use this to avoid errors.